app-localize-behavior.js

```diff
@@ -1,219 +1,214 @@
+ the user can fetch only the needed resources on-demand, and doesn't have to
+ load any resources they'll never see anyway. For example, you could store
+ your resources for your global nav, homepage, and FAQ page in 3 different
+ files. When a user requests the homepage, both the global nav and the
+ homepage resources are fetched and merged together, since they both appear
+ on the page at the same time, but you spare the user from fetching the
+ unneeded FAQ resources.
+
+
+ Example:
+
+     attached: function() {
+       this.loadResources(
+         // Only contains the flattened "es" translations:
+         'locales/es.json',  // {"hi": "hola"}
+         'es',               // unflatten -> {"es": {"hi": "hola"}}
+         true                // merge so existing resources won't be clobbered
+       );
+     }
+
+ Alternatively, you can also inline your resources inside the app itself:
+
+     import {PolymerElement, html} from '@polymer/polymer';
+     import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+     import {AppLocalizeBehavior} from
+ '@polymer/app-localize-behavior/app-localize-behavior.js';
+
+     class SampleElement extends  extends mixinBehaviors(
+         [AppLocalizeBehavior], PolymerElement) {
+       static get template() {
+         return html`
+           <div>{{localize('hello', 'name', 'Batman')}}</div>
+         `;
+       }
+
+       static get properties() {
+         return {
+           language: { value: 'en' },
+           resources: {
+             value: function() {
+               return {
+                 'en': { 'hello': 'My name is {name}.' },
+                 'fr': { 'hello': 'Je m\'appelle {name}.' }
+               }
+           },
+         }
+       }
+
+       function attached() {
+         this.loadResources(this.resolveUrl('locales.json'));
+       }
+     }
+     customElements.define('sample-element', SampleElement);
+
+ @demo demo/index.html
+ @polymerBehavior Polymer.AppLocalizeBehavior
+ */
 export const AppLocalizeBehavior = {
   /**
-   * Internal singleton cache. This is the private implementation of the
-   * behaviour; don't interact with it directly.
+   Internal singleton cache. This is the private implementation of the
+   behaviour; don't interact with it directly.
    */
   __localizationCache: {
-    requests: {},  /* One iron-request per unique resources path. */
-    messages: {},  /* Unique localized strings. Invalidated when the language, formats or resources change. */
-    ajax: null     /* Global iron-ajax object used to request resource files. */
+    requests: {}, /* One iron-request per unique resources path. */
+    messages: {}, /* Unique localized strings. Invalidated when the language,
+                     formats or resources change. */
+    ajax: null    /* Global iron-ajax object used to request resource files. */
   },
 
   /**
-   * Fired after the resources have been loaded.
+   Fired after the resources have been loaded.
    *
```

Comment:
It looks like webmat doesn't like the `*` prefix on comment lines but didn't remove them from the lines where it was the only thing. Those should probably also be removed.